truss/templates/control/control/helpers/truss_patch/model_container_patch_applier.py

```diff
@@ -48,6 +53,15 @@ def __call__(self, patch: Patch):
         elif isinstance(patch.body, SystemPackagePatch):
             sys_pkg_patch: SystemPackagePatch = patch.body
             self._apply_system_package_patch(sys_pkg_patch)
+        elif isinstance(patch.body, ConfigPatch):
```

Comment:
[ignore] i really wish we could switch to python 3.10 and use match statements. https://docs.python.org/3/whatsnew/3.10.html#pep-634-structural-pattern-matching